netket/graph/lattice.py

```diff
@@ -246,6 +301,129 @@ def atoms_coord(self):
         """
         return self._atoms_coord
 
+    def translation_perm(self):
+        perms = []
+        for vec in self.basis_vectors:
```

Comment:
please recall that for loops in python are very expensive and should be avoided as much as possible!